src/Public.Api/Infrastructure/Startup.cs

```diff
@@ -448,6 +438,111 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             return new AutofacServiceProvider(_applicationContainer);
         }
 
+        private static void RegisterExamples(ContainerBuilder containerBuilder)
```

Comment:
FYI types in Api repo:
![image](https://user-images.githubusercontent.com/18366653/222484129-bee3c46c-3aa3-4892-850a-1ca69d665f64.png)